app/mailers/reservation_mailer.rb

```diff
@@ -22,7 +22,29 @@ def car_reservation_approved
     mail(to: @recipients, subject: "Reservation approved for program: #{@reservation.program.display_name}" )
   end
 
-  def car_reservation_cancel(reservation, passengers, emails)
+  def car_reservation_cancel_admin(reservation, passengers, emails)
+    @contact_phone = reservation.program.unit.unit_preferences.find_by(name: "contact_phone").value.presence || ""
+    @passengers = passengers
```

Comment:
@reservation and @passenger is set in before action, you do not need to duplicate this.

Reply:
Since you are passing the reservation and passengers variable in from the call originating in the reservation.rb (model) consider changing the name of the variable(s) to make this code more readable and easier to troubleshoot going forward.

Reply:
done - variables renamed